modules/temp-deck/temp-deck-arduino/temp-deck-arduino.ino

```diff
@@ -134,6 +138,7 @@ PID myPID(&CURRENT_TEMPERATURE, &TEMPERATURE_SWING, &TARGET_TEMPERATURE, DOWN_PI
 
 String device_serial = "";  // leave empty, this value is read from eeprom during setup()
 String device_model = "";   // leave empty, this value is read from eeprom during setup()
+int model_version;          // value is read from device_model during setup()
```

Comment:
It looks like `model_version` is only used in `setup`, it can be a local variable there